public/app/plugins/datasource/prometheus/legend.test.ts

```diff
@@ -18,13 +18,31 @@ describe('renderLegendFormat()', () => {
 
     // not sure if this is expected... but current behavior
     expect(renderLegendFormat('{{ a }}', labels)).toEqual('AAA');
+    expect(renderLegendFormat('{{             a                        }}', labels)).toEqual('AAA');
+  });
+
+  it('Missing label', () => {
+    expect(renderLegendFormat('value: {{c}}', labels)).toEqual('value: {{c}}');
```

Comment:
the output for missing labels changed in this PR. instead of returning the label key, the full match is now returned. this can be reverted, but when adding test coverage it seemed easier to spot issues with label keys when the full match is returned.

```
input: {{c}}
output from main: c
output from this PR: {{c}}
```